aws-rds-dbparametergroup/src/test/java/software/amazon/rds/dbparametergroup/UpdateHandlerTest.java

```diff
@@ -162,6 +167,62 @@ public void handleRequest_SimpleSuccessWithApplyParameters() {
         verify(rdsClient).describeEngineDefaultParameters(any(DescribeEngineDefaultParametersRequest.class));
     }
 
+    @Test
```

Comment:
nit: Can you add the test that verifies that parameters in `modifyDBParameterGroup` are also applied in alphabetical order?